api/src/services/healthService.js

```diff
@@ -0,0 +1,83 @@
+const { blockchainNames, networkNames, accountGroups } = require('../config/blockchains');
+const accountsService = require('./accountsService');
+
+async function checkAccountsBalances(req, res) {  
+  try {
+    const { query } = req;
+    const blockchains = query.blockchains && query.blockchains.split('|');
+    const networks = query.networks && query.networks.split('|');
+    const groups = query.groups && query.groups.split('|');
+    const addresses = query.addresses && query.addresses.split('|');
+    
+    if (!blockchains) {
```

Comment:
1. Move params checking to separate methods.
2. Do params getting one by one. It is no sense to extract addresses if there are no blockchains

Reply:
Fixed